content/Secure_Domain/Introduction.adoc

```diff
@@ -0,0 +1,8 @@
+== Introduction
+
+YubiKeys with firmware version 5.3.0 and later support Secure Channel Protocol 03 (SCP03) for establishing secure communication channels. Starting with firmware version 5.7.2, they also support Secure Channel Protocol 11 (SCP11). For both protocols, secure communication with the YubiKey requires the secure channel to be authenticated using the appropriate certificate chain. Detailed information about the implementation specifics of these protocols can be found in Yubico's technical documentation here.
```

Comment:
"here" looks like it's meant to be a link?

Reply:
good catch! added link.